Develop

IKS-cod opened the pull request.

IKS-cod committed:
Создал шаблон проекта

Commit:
Вставил в pom.xml h2 database и springdoc-openapi

gdsStern committed:
Merge pull request #17 from IKS-cod/TW12

Вставил в pom.xml h2 database и springdoc-openapi

IKS-cod committed:
Сделал задачу TW-7

Создал класс Comment , применил документирующие аннотации Swagger

IKS-cod committed:
Merge branch 'develop' into TW-7

IKS-cod committed:
Сделал задачу TW-7

Создал класс Comment , применил документирующие аннотации Swagger

IKS-cod committed:
Merge pull request #18 from IKS-cod/TW-7

Tw-7

KatrinSkt committed:
Выполнила задачу #TW-4

Создала класс Ad, применила документирующие аннотации Swagger

Commit:
Заполнил шаблон Login, применил документирующие аннотации Swagger

KatrinSkt committed:
Merge pull request #19 from IKS-cod/TW-4

Выполнила задачу #TW-4

gdsStern committed:
Merge pull request #20 from IKS-cod/TW-13

Заполнил шаблон Login, применил документирующие аннотации Swagger

SeregaWinner committed:
Выполнил задание TW1:

1.Создать класс NewPassword;
2.Применил документацию аннотаций Swagger.

IKS-cod committed:
Сделал задачу TW-8

Создал класс Comments , применил документирующие аннотации Swagger

SeregaWinner committed:
Merge pull request #21 from IKS-cod/TW-1

TW1:

IKS-cod committed:
Merge pull request #22 from IKS-cod/TW-8

TW-8

KatrinSkt committed:
Выполнила задачу #TW-5

Создала класс Ads, применила документирующие аннотации Swagger

Commit:
Заполнил шаблон Register, применил документирующие аннотации Swagger

KatrinSkt committed:
Merge pull request #23 from IKS-cod/TW5

Выполнила задачу #TW-5

gdsStern committed:
Merge pull request #24 from IKS-cod/TW-14

Заполнил шаблон Register, применил документирующие аннотации Swagger

KatrinSkt committed:
Выполнила задачу #TW-6

Создала класс ExtendedAd, применила документирующие аннотации Swagger

KatrinSkt committed:
Merge pull request #25 from IKS-cod/TW6

Выполнила задачу #TW-6

IKS-cod committed:
Сделал задачу TW-11

Создал класс CommentController, применил документирующие аннотации Swagger

SeregaWinner committed:
Выполнил задание TW2:

1.Создал класс User;
2.Применил документацию аннотаций Swagger.

SeregaWinner committed:
Merge pull request #26 from IKS-cod/TW-2

TW2:

IKS-cod committed:
Merge pull request #27 from IKS-cod/TW-11

TW-11

SeregaWinner committed:
Выполнил задание TW3:

1.Создал класс UpdateUser;
2.Применил документирующие аннотации Swagger.

SeregaWinner committed:
Merge pull request #29 from IKS-cod/TW-3

TW3:

Commit:
Создал класс CreateOrUpdateComment

IKS-cod committed:
Сделал задачу TW-16

Создал класс CreateOrUpdateAd, применил документирующие аннотации Swagger

gdsStern committed:
Merge pull request #30 from IKS-cod/TW-15

Создал класс CreateOrUpdateComment

IKS-cod committed:
Merge pull request #31 from IKS-cod/TW-16

TW-16

SeregaWinner committed:
Выполнил задание TW9:

1.Создал класс UserController;
2.Применил документирующие аннотации Swagger.

KatrinSkt committed:
Выполнила задачу #TW-10

Создала классы AdController и AdService, применила документирующие аннотации Swagger

SeregaWinner committed:
Merge pull request #32 from IKS-cod/TW-9

TW9:

KatrinSkt committed:
Merge pull request #33 from IKS-cod/TW-10

Выполнила задачу #TW-10

OlgaLugacheva commented on src/main/java/TW2/dto/Ad.java:
```diff
@@ -0,0 +1,97 @@
+
+import java.util.Objects;
+
+public class Ad {
```
Обычно dto классы иак называют AdDto

IKS-cod committed:
Сделал задачу TW-17

Добавил зависимости в pom, заполнил application, переименовать Dto

IKS-cod committed:
Merge pull request #44 from IKS-cod/TW-17

TW-17

SeregaWinner committed:
Выполнил задание TW22:

Создал класс Users;

SeregaWinner committed:
Merge pull request #45 from IKS-cod/TW-22

TW22:

KatrinSkt committed:
Выполнила задачу TW-18

Cоздала класс Ads

KatrinSkt committed:
Merge pull request #46 from IKS-cod/TW-18

Выполнила задачу TW-18

Commit:
выполнил задание TW-19

gdsStern committed:
Merge pull request #47 from IKS-cod/TW-19

выполнил задание TW-19

KatrinSkt committed:
Выполнила задачу TW-23

Создала класс Mappers

IKS-cod committed:
Сделал задачу TW-20

Создал класс Comments

Commit:
выполнил задание TW-21

KatrinSkt committed:
Merge pull request #48 from IKS-cod/TW-23

Выполнила задачу TW-23

IKS-cod committed:
Merge pull request #49 from IKS-cod/TW-20

TW-20

SeregaWinner committed:
Выполнил задание TW24:

1.Добавил liquibase для работы с таблицами в Db;
2.прописал параметры в application.properties.

gdsStern committed:
Merge pull request #50 from IKS-cod/TW-21

выполнил задание TW-21

SeregaWinner committed:
Merge pull request #51 from IKS-cod/TW-24

TW24:

Commit:
выполнил задание TW-25

gdsStern committed:
Merge pull request #52 from IKS-cod/TW-25

выполнил задание TW-25

SeregaWinner committed:
Выполнил задание TW32:

1.Исправил, дополнил UserController;
2.Написал javadoc;
3.Написал логирование.

SeregaWinner committed:
Merge pull request #100 from IKS-cod/TW-32

TW32:

KatrinSkt committed:
Выполнила задачу TW-31

Исправила класс CommentController
Написала javadoc
Добавила логирование

KatrinSkt committed:
Merge pull request #101 from IKS-cod/TW-31

Выполнила задачу TW-31

SeregaWinner committed:
Выполнил задание TW39:

1.Исправил, дополнил ExtendedAdDto;
2.Написал javadoc;
3.Написал логирование.

IKS-cod committed:
Сделал задачу TW-26

Исправил класс WebSecurityConfig, дописал javadoc, логирование

SeregaWinner committed:
Merge pull request #102 from IKS-cod/TW-39

TW39:

IKS-cod committed:
Merge pull request #103 from IKS-cod/TW-26

TW-26

Commit:
выполнил задание TW-62

gdsStern committed:
Merge pull request #104 from IKS-cod/TW-62

выполнил задание TW-62

KatrinSkt committed:
Выполнила задачу TW-36

Исправила класс CommentsDto
Написала javadoc
Добавила логирование

KatrinSkt committed:
Merge pull request #105 from IKS-cod/TW-36

Выполнила задачу TW-36

SeregaWinner committed:
Выполнил задание TW44:

1.Исправил, дополнил UpdateUserDto;
2.Написал javadoc;
3.Написал логирование.

IKS-cod committed:
Сделал задачу TW-27

Исправил класс AdController, дописал javadoc, логирование

SeregaWinner committed:
Merge pull request #106 from IKS-cod/TW-44

TW44:

IKS-cod committed:
Merge pull request #107 from IKS-cod/TW-27

TW-27

Commit:
выполнил задание TW-57

KatrinSkt committed:
Выполнила задачу TW-37

Исправила класс CreateOrUpdateAdDto
Написала javadoc
Добавила логирование

SeregaWinner committed:
Выполнил задание TW45:

1.Исправил, дополнил UserDto;
2.Написал javadoc;
3.Написал логирование.

gdsStern committed:
Merge pull request #108 from IKS-cod/TW-57

выполнил задание TW-57

KatrinSkt committed:
Merge pull request #109 from IKS-cod/TW-37

Выполнила задачу TW-37

SeregaWinner committed:
Merge pull request #110 from IKS-cod/TW-45

TW45:

IKS-cod committed:
Сделал задачу TW-28

Исправил в tw2.sql в таблице comments в колонке createdAt тип на BIGINT

IKS-cod committed:
Merge pull request #111 from IKS-cod/TW-28

TW-28

Commit:
выполнил задание TW-49

gdsStern committed:
Merge pull request #112 from IKS-cod/TW-49

выполнил задание TW-49

IKS-cod committed:
Сделал задачу TW-29

Исправил класс AuthController, дописал javadoc, логирование

IKS-cod committed:
Merge pull request #113 from IKS-cod/TW-29

TW-29

Commit:
выполнил задание TW-48

KatrinSkt committed:
Выполнила задачу TW-38

Исправила класс CreateOrUpdateCommentDto
Написала javadoc
Добавила логирование

KatrinSkt committed:
Merge pull request #115 from IKS-cod/TW-38

Выполнила задачу TW-38

gdsStern committed:
Merge pull request #114 from IKS-cod/TW-48

выполнил задание TW-48

IKS-cod committed:
Сделал задачу TW-33

Исправил класс AdDto, дописал javadoc, логирование

SeregaWinner committed:
Выполнил задание TW50:

1.Написал класс UserNotAuthenticatedException;
2.Написал javadoc;
3.Написал логирование.

IKS-cod committed:
Merge pull request #116 from IKS-cod/TW-33

TW-33

SeregaWinner committed:
Merge pull request #117 from IKS-cod/TW-50

TW50:

Commit:
выполнил задание TW-43

gdsStern committed:
Merge pull request #118 from IKS-cod/TW-43

выполнил задание TW-43

IKS-cod committed:
Сделал задачу TW-34

Исправил класс AdsDto, дописал javadoc, логирование

IKS-cod committed:
Merge pull request #119 from IKS-cod/TW-34

TW-34

KatrinSkt committed:
Выполнила задачу TW-47

Создала класс AvatarNotFoundException
Написала javadoc
Добавила логирование

SeregaWinner committed:
Выполнил задание TW51:

1.Написал класс UserNotFoundException;
2.Написал javadoc;
3.Написал логирование.

KatrinSkt committed:
Merge pull request #120 from IKS-cod/TW-47

Выполнила задачу TW-47

SeregaWinner committed:
Merge pull request #121 from IKS-cod/TW-51

TW51:

IKS-cod committed:
Сделал задачу TW-35

Исправил класс CommentDto, дописал javadoc, логирование

Commit:
выполнил задание TW-42

gdsStern committed:
Merge pull request #122 from IKS-cod/TW-42

выполнил задание TW-42

IKS-cod committed:
Merge pull request #123 from IKS-cod/TW-35

TW-35

SeregaWinner committed:
Выполнил задание TW58:

1.Исправил класс Users;
2.Написал javadoc;
3.Написал логирование.

Commit:
выполнил задание TW-41

KatrinSkt committed:
Выполнила задачу TW-52

Исправила класс BasicAuthCorsFilter
Написала javadoc

SeregaWinner committed:
Merge pull request #124 from IKS-cod/TW-58

TW58:

gdsStern committed:
Merge pull request #125 from IKS-cod/TW-41

выполнил задание TW-41

KatrinSkt committed:
Merge pull request #126 from IKS-cod/TW-52

Выполнила задачу TW-52

Commit:
выполнил задание TW-40

gdsStern committed:
Merge pull request #127 from IKS-cod/TW-40

выполнил задание TW-40

SeregaWinner committed:
Выполнил задание TW63:

1.Исправил класс UsersRepository;
2.Написал javadoc;

IKS-cod committed:
Сделал задачу TW-46

Создал класс AdNotFoundException, написал javadoc, логирование

SeregaWinner committed:
Merge pull request #128 from IKS-cod/TW-63

TW63:

KatrinSkt committed:
Выполнила задачу TW-55

Исправить класс Avatars
Написала javadoc
Добавила логирование

IKS-cod committed:
Merge pull request #129 from IKS-cod/TW-46

TW-46

KatrinSkt committed:
Merge pull request #130 from IKS-cod/TW-55

Выполнила задачу TW-55

IKS-cod committed:
Сделал задачу TW-53

Исправил класс Mappers , написал javadoc, логирование

Commit:
выполнил задание TW-30

gdsStern committed:
Merge pull request #131 from IKS-cod/TW-30

выполнил задание TW-30

IKS-cod committed:
Merge pull request #132 from IKS-cod/TW-53

TW-53

SeregaWinner committed:
Выполнил задание TW64:

1.Исправил, дополнил класс AuthServiceImpl;
2.Написал javadoc;
3.Написал логирование

IKS-cod committed:
Сделал задачу TW-54

Исправил класс Ads , написал javadoc, логирование

SeregaWinner committed:
Merge pull request #133 from IKS-cod/TW-64

TW64:

IKS-cod committed:
Merge pull request #134 from IKS-cod/TW-54

TW-54

IKS-cod committed:
Сделал задачу TW-56

Исправил класс Comments , написал javadoc, логирование

Commit:
выполнил задание TW-69

gdsStern committed:
Merge pull request #135 from IKS-cod/TW-69

выполнил задание TW-69

IKS-cod committed:
Merge pull request #136 from IKS-cod/TW-56

TW-56

KatrinSkt committed:
Выполнила задачу TW-60

Исправить класс AvatarsRepository
Написала javadoc

KatrinSkt committed:
Merge pull request #137 from IKS-cod/TW-60

Выполнила задачу TW-60

IKS-cod committed:
Сделал задачу TW-59

Исправил класс AdsRepository , написал javadoc, логирование

IKS-cod committed:
Merge pull request #138 from IKS-cod/TW-59

TW-59

SeregaWinner committed:
Выполнил задание TW71:

1.Создал класс UserService прописал методы;
2.Написал javadoc;
3.Написал логирование

SeregaWinner committed:
Merge pull request #139 from IKS-cod/TW-71

TW71:

Commit:
выполнил задание TW-66

gdsStern committed:
Merge pull request #140 from IKS-cod/TW-66

выполнил задание TW-66

SeregaWinner committed:
Выполнил задание TW72:

1.Создал класс UserVerification прописал методы;
2.Написал javadoc;
3.Написал логирование

KatrinSkt committed:
Выполнила задачу TW-61

Исправить класс CommentsRepository
Написала javadoc

SeregaWinner committed:
Merge pull request #141 from IKS-cod/TW-72

TW72:

IKS-cod committed:
Сделал задачу TW-65

Исправил класс AdService , написал javadoc, логирование

KatrinSkt committed:
Merge pull request #142 from IKS-cod/TW-61

Выполнила задачу TW-61

IKS-cod committed:
Merge pull request #143 from IKS-cod/TW-65

TW-65

KatrinSkt committed:
Выполнила задачу TW-67

Создала класс AvatarService
Написала javadoc
Добавила логгирование

KatrinSkt committed:
Merge pull request #144 from IKS-cod/TW-67

Выполнила задачу TW-67

IKS-cod committed:
Сделал задачу TW-68

Исправил класс CommentService , написал javadoc, логирование

IKS-cod committed:
Merge pull request #145 from IKS-cod/TW-68

TW-68

KatrinSkt committed:
Выполнила задачу TW-67

Создала класс UserContextService
Написала javadoc
Добавила логирование

KatrinSkt committed:
Merge pull request #146 from IKS-cod/TW-70

Выполнила задачу TW-67

IKS-cod committed:
Сделал задачу TW-73

Создал в корне папку defaultAvatars с файлом java.png и папку image в ней создать папки images и avatars

IKS-cod committed:
Сделал задачу TW-73

Создал в корне папку defaultAvatars с файлом java.png и папку image в ней создать папки images и avatars

IKS-cod committed:
Merge pull request #147 from IKS-cod/TW-73

TW-73

IKS-cod committed:
Сделал задачу TW-74

Исправил файл application.properties

IKS-cod committed:
Merge pull request #150 from IKS-cod/TW-74

TW-74

IKS-cod committed:
Сделал задачу TW-75

Создал класс ValidationService, добавил логирование и описание javadoc

IKS-cod committed:
Merge pull request #152 from IKS-cod/TW-75

TW-75

IKS-cod committed:
Сделал задачу TW-82

Создал application.properties для тестов, с настройками подключения базы данных h2

IKS-cod committed:
Merge pull request #160 from IKS-cod/TW-82

TW-82

KatrinSkt committed:
Выполнила задачу TW-76

Добавила валидацию полей в класс CreateOrUpdateAdDto
Исправила javadoc

KatrinSkt committed:
Merge pull request #161 from IKS-cod/TW-76

Выполнила задачу TW-76

KatrinSkt committed:
Выполнила задачу TW-77

Добавила валидацию полей в класс CreateOrUpdateCommentDto, исправила javadoc

KatrinSkt committed:
Merge pull request #162 from IKS-cod/TW-77

Выполнила задачу TW-77

SeregaWinner committed:
Выполнил задание TW78:

1.Добавил валидацию полей в класс LoginDto;
2.Поправил javadoc;

SeregaWinner committed:
Merge pull request #163 from IKS-cod/TW-78

TW78:

SeregaWinner committed:
Выполнил задание TW81:

1.Добавил валидацию полей в класс UpdateUserDto;
2.Поправил javadoc;

SeregaWinner committed:
Merge pull request #164 from IKS-cod/TW-81

TW81:

Commit:
выполнил задание TW-79

gdsStern committed:
Merge pull request #165 from IKS-cod/TW-79

выполнил задание TW-79

Commit:
выполнил задание TW-80

gdsStern committed:
Merge pull request #166 from IKS-cod/TW-80

выполнил задание TW-80

IKS-cod committed:
Сделал задачу TW-83

Создал класс GlobalExceptionHandler с аннотацией @ControllerAdvice

IKS-cod committed:
Merge pull request #168 from IKS-cod/TW-83

TW-83

IKS-cod committed:
Сделал задачу TW-84

Создал класс AuthControllerTest и написал тесты

IKS-cod committed:
Merge pull request #170 from IKS-cod/TW-84

TW-84

IKS-cod committed:
Сделал задачу TW-85

Добавил тесты

IKS-cod committed:
Merge pull request #171 from IKS-cod/TW-85

TW-85

IKS-cod committed:
Сделал задачу TW-86

Изменил метод updateAdImage

IKS-cod committed:
Merge pull request #173 from IKS-cod/TW-86

TW-86

IKS-cod committed:
Сделал задачу TW-87

Изменил доступ к вызову эндпоинтов

IKS-cod committed:
Merge pull request #177 from IKS-cod/TW-87

TW-87

IKS-cod committed:
Сделал задачу TW-89

Изменил класс ImageService метод updateAdImage

IKS-cod committed:
Merge pull request #179 from IKS-cod/TW-89

TW-89

IKS-cod committed:
Сделал задачу TW-90

Обработка ошибки аутентификации

IKS-cod committed:
Merge pull request #181 from IKS-cod/TW-90

TW-90

IKS-cod committed:
Сделал задачу TW-91

Рефакторинг тестов

IKS-cod committed:
Merge pull request #183 from IKS-cod/TW-91

TW-91

SeregaWinner committed:
Выполнил задание TW92:

Рефакторинг кода UserController и AdController;

SeregaWinner committed:
Merge pull request #185 from IKS-cod/TW-92

TW92:

IKS-cod committed:
Сделал задачу TW-94

Написал тесты для AvatarAndImageControllerTest

IKS-cod committed:
Merge pull request #189 from IKS-cod/TW-94

TW-94

OlgaLugacheva commented on src/test/java/TW2/controller/UserControllerTest.java:
```diff
@@ -0,0 +1,7 @@
+import static org.junit.jupiter.api.Assertions.*;
+
+class UserControllerTest {
+
```
класс есть а тестов нет

OlgaLugacheva commented on src/main/java/TW2/service/ValidationService.java:
```diff
@@ -0,0 +1,86 @@
+    public static boolean isValidLength(String name, int min, int max) {
+        boolean isValid = name.length() >= min && name.length() <= max;
+        logger.log(Level.INFO, "Проверка длины: {0} (min: {1}, max: {2}) - Результат: {3}",
+                new Object[]{name, min, max, isValid});
```
В принципе валидацию можно организовать аннотациями, даже если если это какая-то специальная валидация - можно написать свою аннотацию

IKS-cod committed:
Сделал задачу TW-100

Написал тесты для UserControllerTest и AdControllerTest

IKS-cod committed:
Merge pull request #191 from IKS-cod/TW-100

TW-100

SeregaWinner committed:
Выполнил задание TW99:

Написал тесты на сервисы.

SeregaWinner committed:
Merge pull request #193 from IKS-cod/TW-99

TW99:
